src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day--m calendar--month-length--31">
```

Comment:
`calendar--start-day--m` you dont have modificator like `m` it must be `mon` or one of six else.

Reply:
Solved.